chinglee-iot/FreeRTOS
# Coverage tests for xTaskPriorityDisinherit, and vTaskPriorityDisinheritAftertimeout

Description
-----------
Coverage tests for xTaskPriorityDisinherit, and vTaskPriorityDisinheritAftertimeout

Test Steps
-----------
cd FreeRTOS/Test/CMock/smp
rm -rf ../build/generated
make lcovhtml

view the resulting generated file with a web browser: FreeRTOS/Test/CMock/build/generated/smp/coverage/Source/tasks.c.gcov.html


Related Issue
-----------
TS-24213, TS-24214


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
